File: LogbookEntry+TDate.m
//
//  LogbookEntry+TDate.m
//  GymBuddy
//
//  Created by John Neyer on 2/19/12.
//  Copyright (c) 2012 jneyer.com. All rights reserved.
//

#import "LogbookEntry+TDate.h"

@implementation LogbookEntry (TDate)

-(NSDate *) getDate_t
{
    NSDateFormatter *format = [[NSDateFormatter alloc] init];
    [format setDateFormat:@"dd MMM yyyy"];
    NSString *dateOnly = [format stringFromDate: self.date];

    return [format dateFromString:dateOnly]; 
}

@end

File: LogbookEntry+TDate.swift
//
//  LogbookEntry_TDate.swift
//  FitBuddy
//
//  Created by John Neyer on 4/29/15.
//  Copyright (c) 2015 jneyer.com. All rights reserved.
//

import Foundation


extension LogbookEntry {
    
    class var date_t : NSDate {
    
        let format: NSDateFormatter = NSDateFormatter.new()
        format.dateFormat = "dd MMM yyyy"
        
        let dateOnly = format.stringFromDate(self.valueForKey("date") as! NSDate)
        return format.dateFromString(dateOnly)!
 
    }
    
}